mlir/lib/Dialect/Vector/Transforms/LowerVectorGather.cpp

```diff
@@ -55,6 +55,8 @@ namespace {
 /// ```
 ///
 /// When applied exhaustively, this will produce a sequence of 1-d gather ops.
+///
+/// Supports vector types with trailing scalable dim.
```

Comment:
nit: It supports vector types with a leading fixed dim (the trailing dim does not really matter).